src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let result = {};
+
+  for (const i in robot) {
```

Comment:
```suggestion
  for (const key in robot) {
```
i хорошее название для итератора, но плохое для ключа объекта